src/scripts/run_certbot.sh

```diff
@@ -116,7 +116,7 @@ done
 
 # Iterate over each key and make a certificate request for them.
 for cert_name in "${!certificates[@]}"; do
-    server_names=(${certificates["$cert_name"]})
+    server_names=("${certificates["$cert_name"]}")
```

Comment:
I am sceptical about these quotes as well, since I believe this is what happens:

```bash
server_names=(${certificates["$cert_name"]}) -> server_names=(a b c) => [a, b, c]
```

while the suggested edit would become

```bash
server_names=("${certificates["$cert_name"]}") -> server_names=("a b c") => ["a b c"]
```

This should probably also be tested in some minimal example.

Reply:
https://stackoverflow.com/a/27701642 this leads me to believe that we're safe. The quotes protect us from accidental word splitting and other issues which seem hard to find. I'd wager that if we need word splitting it's better to be explicit than implicit. But I am learning as well so I might be wrong :smile: 

This applies to our other discussions around quoting variables as well.